test/SmokeTest.Tests/HighLevelTest.cs

```diff
@@ -71,6 +96,15 @@ private OpaClient GetOpaClient()
     return new OpaClient(serverUrl: requestUri.ToString());
   }
 
+  private OpaClient GetOpaClientWithLogger(ILogger<OpaClient> logger)
+  {
+    // Construct the request URI by specifying the scheme, hostname, assigned random host port, and the endpoint "uuid".
+    var requestUri = new UriBuilder(Uri.UriSchemeHttp, _containerOpa.Hostname, _containerOpa.GetMappedPublicPort(8181)).Uri;
+
+    // Send an HTTP GET request to the specified URI and retrieve the response as a string.
+    return new OpaClient(serverUrl: requestUri.ToString(), logger: logger);

```

Comment:
Pretty sure this comment does not belong to the accompanying line of code.

Reply:
That's definitely a lingering copy/paste from an earlier era. Good catch. :slightly_smiling_face: